src/ui/pages/BridgeForm/ZerionFeeLine/ZerionFeeLine.tsx

```diff
@@ -37,12 +37,12 @@ export function ZerionFeeLine({ quote }: { quote: Quote | null }) {
             target="_blank"
           >
             <QuestionIcon
-              role="decoration"
+              role="presentation"
```

Comment:
changed to presentation, idk what that "decoration" role means

Reply:
Awesome thanks